docs/superdesk-ninjs-schema.json

```diff
@@ -382,7 +382,15 @@
 					},
 					"jobtitle" : {
 						"description" : "author's Job Title qcode",
-						"type" : "string"
+						"type" : "object",
+						"properies" :  {
+							"qcode": {
```

Comment:
we use code for the other properties, would be good imo to be consistent.
if there is also scheme it should go into `scheme` field, otherwise just `code`